gtsam/inference/inferenceExceptions.h

```diff
@@ -12,30 +12,65 @@
 /**
  * @file    inferenceExceptions.h
  * @brief   Exceptions that may be thrown by inference algorithms
- * @author  Richard Roberts
+ * @author  Richard Roberts, Varun Agrawal
  * @date    Apr 25, 2013
  */
 #pragma once
 
 #include <gtsam/global_includes.h>
+
 #include <exception>
+#include <sstream>
```

Comment:
This will now be included in *all* files that include these exceptions. I propose moving to a cpp file

Reply:
Done